Week2/homework/exercise-3.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/*****Ever wondered how to make a certain meal? Let's create a recipe list with JavaScript!
+
+Declare a variable that holds an object (your meal recipe).
+Give the object 3 properties: a title (string), a servings (number) and an ingredients (array of strings) property.
+Log each property out seperately, using a loop (for, while or do/while)
+It should look similar to this:
+
+Meal name: Omelete
+Serves: 2
+Ingredients:
+4 eggs
+2 strips of bacon
+1 tsp salt/pepper******/
+
+const myMeal = {
+  'Meal name': 'corba',
+  serves: 6,
+  ingredients: ['2 tomato', '1 cup flavor', '2 cup water'],
```

Comment:
👍 but a suggestion to keep all keys inside single quotes for consistency if you have one key like `'Meal name'` inside quotes.